test/Modules/PBWModules.jl

```diff
@@ -0,0 +1,61 @@
+@testset "modules over PBWAlgQuo" begin
+    #Free Module tests
+    E,x = exterior_algebra(QQ, 3)
```

Comment:
This function is part of experimental and should thus not be used in the tests for `src/` code

Reply:
True. Though maybe we need to look into promoting exterior_algebra one of these days?

In any case, note that this PR is still a very early draft, I mainly asked @Lax202 to open it so we can better see what she's doing and what the state is, and then discuss it. The code still needs to be consolidated into a new source file, and that may very well ultimately end up in the `experimental` tree. We'll see.

Reply:
In the meantime this was merged as part of PR #3988 which copied the tests from here. If we still don't want the `exterior_algebra` call here, then someone else should remove it from there.